configuration/concepts/commons.csv

```diff
@@ -2,4 +2,8 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
 67218dc9-aee7-40d1-ba78-d29dd2731333,,CIEL:1362,One plus,+,Uno más,+,,Misc,N/A,,,,,,,,,,,
 55800eb1-4e0f-4a75-a12b-aa1e10c64e81,,CIEL:1363,Two plus,++,Dos más,++,,Misc,N/A,,,,,,,,,,,
 17327c32-0a60-4f76-85b5-24379fc72ad5,,CIEL:1364,Three plus,+++,Tres más,+++,,Misc,N/A,,,,,,,,,,,
-f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
\ No newline at end of file
+f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
+95d3e6b9-1286-4fcb-b45c-fd4689593090,,,Minutes,Min,Minutos,Minutos,,Misc,N/A,,,,,,,,,,,
+c07cffc7-608c-4a47-9370-de2373f82966,,,Months,Months,Meses,Meses,,Misc,N/A,,,,,,,,,,,
+afaa9819-e6fd-40d4-8e53-70c0c762b73b,,,Years,Yrs,Años,Años,,Misc,N/A,,,,,,,,,,,
+72335bf5-566a-40cb-8ea0-e0014a6377ec,,,Weeks,Wks,Semanas,Semanas,,Misc,N/A,,,,,,,,,,,
```

Comment:
These concepts already exist...See [here](https://github.com/globalhealthcoalition/openmrs-config-ghc/blob/master/configuration/concepts/others.csv#L2-L7)
Let us not duplicate them

Reply:
Noted